src/pages/ChatPage/components/BlockUserDialog.tsx

```
@@ -0,0 +1,74 @@
+            className="focus:ring-offset-background rounded-xl bg-error py-3.5 font-medium text-error-foreground transition-colors hover:bg-error/90 focus:outline-none focus:ring-2 focus:ring-error/50 focus:ring-offset-2"
+          >
+            {t("block.confirm", "Yes, Block")}
+          </button>
+
+          <button
+            type="button"
+            onClick={onCancel}
+            role="dialog"
+            aria-modal="true"
+            className="text-foreground border-border focus:ring-offset-background rounded-xl border bg-transparent py-3.5 font-medium transition-colors hover:bg-muted focus:outline-none focus:ring-2 focus:ring-muted-foreground/50 focus:ring-offset-2"
+          >
```

Biome (2.3.13)

[error] Interactive elements should not be assigned non-interactive roles.

WAI-ARIA roles should not be used to convert an interactive element to a non-interactive element.
Wrap your interactive element in a <div> with the desired role or put the content inside your interactive element.
Unsafe fix: Remove the role attribute.


(lint/a11y/noInteractiveElementToNoninteractiveRole)